src/App.tsx

```diff
@@ -47,9 +47,9 @@ function App() {
   return (
     <SourceTargetContextProvider connections={connections}>
       <KeyringContextProvider>
-        <ApiCallsContextProvider>
-          <AccountContextProvider>
-            <SubscriptionsContextProvider>
+        <SubscriptionsContextProvider>
+          <ApiCallsContextProvider>
+            <AccountContextProvider>
               <MessageContextProvider>
```

Comment:
how is that related?

Reply:
As we are using useSubscriptionsContext() inside the `AccountContextProvider` the prop chain inheritance needs to be changed moving the `SubscriptionsContextProvider` upper to the `AccountContextProvider`

Reply:
Seems super fragile - I see basically every 2nd PR changing the order of these.

Reply:
As i'm using the hooks from the contexts and this is a refactor it is expected that this might change. At the moment is under control.